Lib/calendar.py

```diff
@@ -38,8 +42,36 @@ def __str__(self):
 
 
 # Constants for months referenced later
-January = 1
-February = 2
+class Months(IntEnum):
+    __order__ = "JANUARY FEBRUARY MARCH APRIL MAY JUNE JULY AUGUST SEPTEMBER OCTOBER NOVEMEMBER DECEMBER" 
```

Comment:
`__order__` is only required for compatibility with Python 2 code, since that didn't have `__prepare__()`.